File: go-sdk/pkg/models/operations/walletsgetserverinfo.go
package operations

import (
	"github.com/speakeasy-sdks/formance-sdks/go-client-sdk/pkg/models/shared"
)

type WalletsgetServerInfoResponse struct {
	ContentType          string
	ServerInfo           *shared.ServerInfo
	StatusCode           int
	WalletsErrorResponse *shared.WalletsErrorResponse
}

File: ts-sdk/src/sdk/models/operations/walletsgetserverinfo.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";


export class WalletsgetServerInfoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  serverInfo?: shared.ServerInfo;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  walletsErrorResponse?: shared.WalletsErrorResponse;
}